charts/kong/Chart.yaml

```diff
@@ -10,7 +10,7 @@ maintainers:
   email: traines@konghq.com
 name: kong
 sources:
-version: 2.10.2
+version: 2.10.3
```

Comment:
Please strip this change, there are several PRs in flight at the moment and it'll be easier for us to release all their changes at once.